### DISCUSSION TITLE
Potential Coverage by the Python Bytes Podcast

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#636] [Announcements]
I've reached out to one of the hosts for the [Python Bytes Podcast](https://pythonbytes.fm/) and there was some interest in covering IDOM. I feel that once the [last major section of documentation](https://github.com/idom-team/idom/issues/596) is complete, the project will be in a state where it can be announced to a broader audience. With that said, I'd like some feedback on documentation, features, or fixes that should be complete in order to make this public showing as successful as possible. After some time, I'll collect ideas from this discussion into an issue that will capture the major tasks to complete before IDOM's showing on the podcast.

**Comment by rmorshea** [comment]
There should also be a top level "Related Projects" section that links out to the various repos in this org. The projects linked there should be ready to use and reasonably documented.

**Reply by Archmonger** [reply]
Let me know if the podcast plans solidify. I can work on Django-IDOM documentation prior to the appearance.

**Reply by rmorshea** [reply]
I'm going to let the hosts know when they should cover it. I suspect that any coverage would come 1+ months after I give them the go-ahead.
